editor/plugins/theme_editor_plugin.cpp

```diff
@@ -189,11 +189,9 @@ void ThemeEditor::_save_template_cbk(String fname) {
 	}
 
 	FileAccess *file = FileAccess::open(filename, FileAccess::WRITE);
-	if (!file) {
 
-		ERR_EXPLAIN(TTR("Can't save theme to file:") + " " + filename);
-		return;
-	}
+	ERR_FAIL_COND_MSG(!file, "Can't save theme to file: " + filename + ".");
```

Comment:
Just for the reference, the previous code was not printing the error, but it was a bug which is fixed here.